src/cdk/overlay/position/cover-position-strategy.ts

```diff
@@ -0,0 +1,608 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {coerceCssPixelValue} from '@angular/cdk/coercion';
+import {Platform} from '@angular/cdk/platform';
+import {ViewportRuler} from '@angular/cdk/scrolling';
+import {Subscription} from 'rxjs';
+
+import {OverlayReference} from '../overlay-reference';
+
+import {
+  BoundingBox,
+  boundingBoxClass,
+  BoundingBoxRect,
+  clearBoundingBoxStyles,
+  extendStyles,
+  FlexibleConnectedPositionStrategyOrigin,
+  getOriginRect,
+  resetBoundingBoxStyles,
+  resetOverlayElementStyles
+} from './flexible-positioning';
+import {PositionStrategy} from './position-strategy';
+
+/**
+ * A strategy for positioning overlays. Using this strategy, an overlay is given an
+ * implicit position relative some origin element. The relative position is defined in terms of
+ * a point on the origin element that is connected to a point on the overlay element. For example,
+ * a basic dropdown is connecting the bottom-left corner of the origin to the top-left corner
+ * of the overlay.
```

Comment:
Could you include the word "cover" somewhere in this description? I'd expect to see something like
```txt
Strategy for positioning an overlay such that it covers one or more other elements.
You can define the covered region by...
```